### DISCUSSION TITLE
E' possibile anonimizzare la voce in caso di registrazione di messaggi vocali?

### DISCUSSION THREAD

**Discussion by etrix80** [globaleaks/globaleaks-whistleblowing-software#3905] [Q&A] [answered]
Salve, volevo sapere se era prevista implementazione di un sistema di offuscamento/anonimizzazione della voce in caso di segnalazione vocale.
Grazie
Saluti

**Comment by rglauco** [comment, marked as answer]
Buonasera, Globaleaks anonimizza la segnalazione vocale by default
